client/src/components/shared/header/Header.js

```diff
@@ -4,21 +4,15 @@ import { Grid, makeStyles, Button } from '@material-ui/core'
 import routeDict from './RouteDict'
 
 export default function Header() {
-  // ~ Background Color Change ~
-  // This retrieves the route of our current page through useLocation().
-  // Accessed by headerBackgroundColor()
-  const location = useLocation();
-  const route = location.pathname
+  // Get current route
+  const route = useLocation().pathname;
 
-  // This function takes in a route and returns its background color
-  // value located in routeDict.
-  const headerBackgroundColor = (route) => {
-    // Here we check to see if the passed in route is a key in routeDict
-    // If the route key exists, we return its value
+  // Returns the header color based on current route
+  const getHeaderColor = () => {    
```

Comment:
A good naming convention for methods is to use verbs. They describe the action being performed for easier comprehension. This should have been getHeaderBackgroundColor though, oops.

Don't be afraid to use descriptive method names, even if it has to be 4-5 words long.